packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -1,9 +1,9 @@
 <template>
     <div
         ref="megaModal"
-        :class="['c-megaModal', $style['c-megaModal'], {
-            [$style['u-overlay']]: showOverlay
-        }]"
+        :class="['c-megaModal',
+                 { 'u-overlay': showOverlay },
+                 $style['c-megaModal']]"
```

Comment:
I think this was clearer before as you see the standard class and the CSS module class side-by-side? 

It was also following the style used in the Vue docs where you define your classes first followed by your condition classes.